### DISCUSSION TITLE
regarding the sudden drop of the validation loss

### DISCUSSION THREAD

**Discussion by guoqiang-zhang-x** [KellerJordan/modded-nanogpt#113] [Q&A]
I have a question regarding the validation loss for the code version modded-nanogpt-casted. It is seen from the picture that the validation loss for a particular setup first decreases smoothly and then suddenly drops by a large margin.  Is it because of the learning rate scheduling at the end of training? Or the learning rate remains constant after warming up and the validation loss simply drops at some point?  

 The reason I am asking this question is when I test my new training method, I also observe that the validation loss drops at some point with constant learning rate after warming up. I want to make sure my understanding of the picture mentioned above is correct.
